proxies/live/apiproxy/policies/KeyValueMapOperations.GetVariables.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<KeyValueMapOperations async="false" continueOnError="false" enabled="true" name="KeyValueMapOperations.GetVariables" mapIdentifier="pds-variables">
```

Comment:
Is this policy supposed to be here, and should we really be using PDS variables?